server/datastore/mysql/software.go

```diff
@@ -2740,7 +2741,9 @@ func hostVPPInstalls(ds *Datastore, ctx context.Context, hostID uint, globalOrTe
                 (ua2.priority < ua.priority OR ua2.created_at > ua.created_at)
 			LEFT JOIN
 				vpp_apps_teams vat ON vaua.adam_id = vat.adam_id AND vaua.platform = vat.platform AND vat.global_or_team_id = :global_or_team_id
-            WHERE
+			INNER JOIN
+				vpp_apps ON vaua.adam_id = vpp_apps.adam_id
```

Comment:
These need to join on adam_id and platform; GitOps will actually create a `vpp_apps` entry for each of the three associated platforms (iOS, iPadOS, macOS) when you specify an app ID, so if we just join on adam_id we'll get a subtle bug where there are more software titles than there should be.